packages/@best/api-db/src/sql/postgres/migrations/001-organizations.ts

```diff
@@ -6,14 +6,14 @@
 */
 
 export const up = `
-CREATE TABLE projects (
+CREATE TABLE organizations (
```

Comment:
Since this is a database migration, shouldn't we add a new `migrations/002-organizations.ts` file rather than update the existing file? Otherwise the change won't actually be made to existing databases.

Reply:
@nolanlawson organizations should be created first and then projects and then snapshots, as snapshots references projects and projects references organization. For existing projects, we need to manually create organizations and update projects.

Reply:
@sutturu You generally don't want to change the existing migration files since current best projects are using the old schema and you will lose track of how to migrate to and from specific versions. 

The goal of the migration file is to take the old schema before your changes, and write the sql commands that will take the old schema to your new schema where organizations are the top level object.

Also be sure to include a migration for sqlLite as well.